tests/ui/manual_flatten.rs

```diff
@@ -106,4 +106,24 @@ fn main() {
     for n in vec![Some(1), Some(2), Some(3)].iter().flatten() {
         println!("{}", n);
     }
+
+    // Comment below explains this silly struct and function name
+    struct LongConstructorForSomeArray {}
+    impl LongConstructorForSomeArray {
+        fn get_some___________________________________________________________________(
+            p1: Option<i32>,
+        ) -> Vec<Option<i32>> {
+            vec![p1]
+        }
+    }
+
+    // Make sure length of the next line with arguments is above 120, so the
+    // suggestion provided by clippy cannot be shown inline within the error message
+    for n in LongConstructorForSomeArray::get_some___________________________________________________________________(
```

Comment:
I'm really not sure if there is some smarter method to achieve multi-line suggestion here.
My solution is simply `bruteforce` in a way.

Reply:
Ah right!
Let's rather use  `#[rustfmt::skip]`, it feels like more elegant solution here.
Thanks for the suggestion!

Reply:
Yep that is good thanks, a single commit is usually preferred